src/User32.Desktop/User32.cs

```diff
@@ -156,6 +177,12 @@ public static partial class User32
         [return: MarshalAs(UnmanagedType.Bool)]
         public static extern bool MessageBeep(MessageBeepType uType);
 
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern MessageBoxResult MessageBox(IntPtr hWnd, string text, string caption, MessageBoxOptions options);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern int MessageBoxIndirect(ref MSGBOXPARAMS lpMsgBoxParams);
```

Comment:
Should we avoid all ref and out parameters and use pointers instead or there is some rules I could follow?